### ISSUE TITLE
add CIFAR10 tutorial in PyTorch

### ISSUE THREAD

**Issue by npapernot** [tensorflow/cleverhans#1024]

**Comment by jvmancuso** [comment]
I wrote something like this awhile back [here](https://github.com/jvmancuso/PytorchAdversarialGym/blob/master/examples/demoFGSM.ipynb).  The underlying codebase is quite a bit different from cleverhans, but maybe some of that can be reused?

**Comment by michaelshiyu** [comment]
BTW, the current mnist_tutorial in #1028 is only a few edits away from a cifar10 tutorial. Just change torchvision.datasets.MNIST to torchvision.datasets.CIFAR10 and change the in_channel of the model from 1 to 3.

Or do we want a bigger/better network instead of LeNet-5 for this?

Thanks.

**Comment by jvmancuso** [comment]
That would probably make more sense, although I'm not sure what having multiple tutorials with only a few lines' difference would accomplish for a new user.  Perhaps there's functionality missing from the MNIST tutorial that would be useful to add?

**Comment by michaelshiyu** [comment]
Yes I agree that these two tutorials would look similar. And I can't see how a cifar10 tutorial would be able to demonstrate any functionality missing from the existing mnist tutorial.

I think the original idea was to add a cifar10 tutorial but I added a mnist tutotial with my PR instead.

**Comment by npapernot** [comment]
Let's change the dataset from MNIST to CIFAR10 once the PR is merged. The idea was to first get an example for MNIST quickly to help contributors waiting on an API example. Afterwards, we'd like to move away from MNIST since it is usually not an interesting dataset to study adversarial examples on (except in some corner cases).
